### DISCUSSION TITLE
Gnd screen > not audio in or copycat options

### DISCUSSION THREAD

**Discussion by frixid** [eh2k/squares-and-circles#85] [Q&A]
The engine screen is empty

![289C2A53-0490-485B-A2F1-2C3932E29C73](https://user-images.githubusercontent.com/6703830/194781245-2a68287f-c9b2-4164-b1b3-cd0acdb9dd5d.jpeg)

**Comment by eh2k** [comment]
Hi @frixid, the engines you saw in the video were there once, but were never released because the function is given elsewhere, and otherwise everything would be too complicated because of redundancy. For example, you can route audio as AUX input to any FX engine via IO settings. In the past, this was a separate engine that passed the CV directly as audio input to the neighbouring engine.  

I admit the GND/--- is a bit confusing, I have taken it over from Elektron without giving it much thought. Maybe it would be more user-friendly as CV/GND.
